scripts/backport.py

```diff
@@ -289,6 +278,28 @@ def __init__(self, pygithub_pr_, issue_number_):
     prs_to_backport[pull.number].pygithub_commits.insert(0, pygithub_commit)
 
 
+def report_backport_failure(original_pr, reason):
+    """If something prevents us from backporting the PR automatically,
+    report it in a comment to original pr, and add a label preventing
+    further attempts."""
+    print(
+        f"Will not backport the PR #{original_pr.number} '{original_pr.title}': {reason}"
+    )
+
+    github_comment = f"Automated backport to {backport_target} failed: {reason}."
```

Comment:
Not doing the backport might be for many reasons, which does not necessarily means it failed, just that the decision was to not do it.
```suggestion
    github_comment = f"Automated backport to {backport_target} not done: {reason}."
```

Reply:
Done.